Library/Formula/algernon.rb

```diff
@@ -0,0 +1,54 @@
+class Algernon < Formula
+  desc "HTTP/2 web server with Lua support"
+  homepage "http://algernon.roboticoverlords.org/"
+  url "https://github.com/xyproto/algernon/archive/0.74.tar.gz"
+  sha256 "1341af6864643a968d85bfa63ca231604b6d1123919c6826ae179908c6c4a176"
+  head "https://github.com/xyproto/algernon.git"
+
+  depends_on "go" => :build
+  depends_on :hg => :build
+  depends_on "readline"
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    system "go", "get", "-d"
+    system "go", "build", "-o", "algernon"
+
+    bin.install "algernon"
+  end
+
+  test do
+    begin
+      tempdb = "/tmp/_brew_test.db"
+      cport = ":45678"
+
+      # Start the server in a detached process
+      fork_pid = fork do
+        `#{bin}/algernon --httponly --server --addr #{cport} --boltdb #{tempdb}`
```

Comment:
I am capturing the output, in order to hide it. When using the system call, there is quite a bit of unneeded output. I tried using `spawn` as well, to hide the output on stderr and stdout, but that does not work on Ruby 1.8 / Mountain Lion. Is using `system` instead of hiding the output preferred?